docs/contract.yaml

```diff
@@ -0,0 +1,156 @@
+openapi: 3.0.0
+info: 
+  title: Spec Math API Server
+  version: 1.0.0
+  description: An API that exposes some operations from the Spec Math Library
+servers:
+  - description: Spec Math API Server
+    url: http://specmath.cloud.google.com/v1
+paths:
+  /specMath/merge:
```

Comment:
As a test API, this is fine, but I'm uncomfortable with this as a product.

First, this and other endpoints listed here are essentially RPCs and not RESTful, so this design conflicts with most current API design guidance, including [Google's](https://google.aip.dev/121). We may decide to not follow that guidance here, but we should be aware of that.

Next, the "specMath" part of the path subjectively strikes me as awkward - its camel case capitalization seems hard to predict (but if this matches a broader set of patterns that could be ok) - and "specMath" feels more like a brand for something than an actual description. "specmath" as the name of the API seems ok, but here I'd like to have something more descriptive - like "operations" - which would allow for other top-level names in the future.